debezium-core/src/main/java/io/debezium/pipeline/notification/InitialSnapshotNotificationService.java

```diff
@@ -35,35 +36,52 @@ public InitialSnapshotNotificationService(NotificationService<P, O> notification
         this.clock = clock;
     }
 
+    public <T extends DataCollectionId> void notifyTableInProgress(P partition,
+                                                                   OffsetContext offsetContext,
+                                                                   String currentCollection) {
+        notificationService.notify(buildNotificationWith(SnapshotStatus.TABLE_SCAN_IN_PROGRESS.name(),
+                Map.of(SCANNED_COLLECTION, currentCollection)),
+                Offsets.of(partition, offsetContext));
+
+    }
+
+    public <T extends DataCollectionId> void notifyCompletedTable(P partition,
+                                                                  OffsetContext offsetContext,
+                                                                  String currentCollection) {
+        notificationService.notify(buildNotificationWith(SnapshotStatus.TABLE_SCAN_COMPLETED.name(),
+                Map.of(SCANNED_COLLECTION, currentCollection)),
```

Comment:
It is possible to add information about the list of table to be scanned and also `total rows scanned` as for incremental snapshot notification?